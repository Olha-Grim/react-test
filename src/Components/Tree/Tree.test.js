import React from "react";
import { render } from "@testing-library/react";
import { Tree } from ".";

describe("Tree", () => {
  test("interaction with Tree component", () => {
    const { asFragment } = render(
      <Tree className="draggable-tree" draggable blockNode />
    );
    expect(asFragment(<Tree className="draggable-tree"/>)).toMatchSnapshot();
  });
});
