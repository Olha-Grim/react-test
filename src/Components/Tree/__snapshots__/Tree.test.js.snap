// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tree interaction with Tree component 1`] = `
<DocumentFragment>
  <div
    class="ant-tree ant-tree-icon-hide ant-tree-block-node"
  >
    <div
      role="tree"
    >
      <input
        style="width: 0px; height: 0px; display: flex; overflow: hidden; opacity: 0; border: 0px; padding: 0px; margin: 0px;"
        tabindex="0"
        value=""
      />
    </div>
    <div
      aria-hidden="true"
      class="ant-tree-treenode"
      style="position: absolute; pointer-events: none; visibility: hidden; height: 0px; overflow: hidden;"
    >
      <div
        class="ant-tree-indent"
      >
        <div
          class="ant-tree-indent-unit"
        />
      </div>
    </div>
    <div
      class="ant-tree-list"
      style="position: relative;"
    >
      <div
        class="ant-tree-list-holder"
      >
        <div>
          <div
            class="ant-tree-list-holder-inner"
            style="display: flex; flex-direction: column;"
          />
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
